resources/help.txt

```diff
@@ -411,45 +411,78 @@ Options:
         You cannot use --device and --emulator simultaneously.
     --release - If set, produces a release build. Otherwise, produces a debug build.
 --[/]--
-    
+
 --[debug]--
+Usage:
```

Comment:
Well... By default we have help only for commands that exist. In case you write:
$ tns debug
we'll show error message that such command does not exist:

But in case you write $ tns debug -h we'll not validate the command, we'll directly check help.txt for such 'article'. And as you have left it here, we'll show help and when the customer tries ```$ tns debug ``` the command will fail.
I'm not sure which one is better solution. We are working hard on improving our help at the moment and I'll consider this case, but for the moment I prefer to remove it. Ping @teobugslayer , @Fatme  - what do you think about this?